cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/DatabasesTest.scala

```diff
@@ -82,6 +82,7 @@ class DatabasesTest extends DocumentingTest with QueryStatisticsTestSupport {
     }
     section("Creating databases", "administration-databases-create-database", "enterprise-edition") {
       p("Databases can be created using `CREATE DATABASE`.")
+      p("include::create-databases-syntax.asciidoc[]")
```

Comment:
I didn't manage to build it so maybe I missed something, but do we even have a file named like this?

Reply:
I think I thought about adding individual syntax but ended up having a single include at the top in the end.